k8s/dataverse/persona/nfdi4health/init.sh

```diff
@@ -47,7 +47,8 @@ while IFS= read -r TSV; do
 done <<< "${TSVS}"
 
 echo "Disable custom terms of use"
-curl -s -H "X-Dataverse-key:$API_TOKEN" -X PUT -d false $DATAVERSE_URL/aapi/admin/settings/:AllowCustomTermsOfUse
+curl -s -H "X-Dataverse-key:$API_TOKEN" -X PUT -d false "${DATAVERSE_URL}/api/admin/settings/:AllowCustomTermsOfUse"
+echo
```

Comment:
empty `echo`

Reply:
This is for formatting, to create a new line consistent with the other steps in the script